WordPress/Classes/WordPress.xcdatamodeld/.xccurrentversion

```diff
@@ -3,6 +3,6 @@
 <plist version="1.0">
 <dict>
 	<key>_XCCurrentVersionName</key>
-	<string>WordPress 154.xcdatamodel</string>
+	<string>WordPress 155.xcdatamodel</string>
```

Comment:
let's remember to update this pre-production model with new namespace names.
cc @mokagio pining just in case – I think you saw it already.